### PR TITLE
Minor fix found in Kconfig

### PR DESCRIPTION
**Pull request by xiaoxiang781216**
### Description of the feature or fix

Please read the commit message for each fix

### Checkpoints
- [x] Follow the [styling guide](https://github.com/lvgl/lvgl/blob/master/docs/CODING_STYLE.md)
- [ ] Update CHANGELOG.md
- [ ] Update the documentation

### CODE REVIEW

File: Kconfig [hunk 1]
```diff
@@ -1,11 +1,6 @@
 # Kconfig file for LVGL v7.8.1
 
 menu "LVGL configuration"
-    
-    config LV_ATTRIBUTE_FAST_MEM_USE_IRAM
-        bool "Set IRAM as LV_ATTRIBUTE_FAST_MEM"
-        help
-            Set this option to configure IRAM as LV_ATTRIBUTE_FAST_MEM
```
Comment (liebman) [inline Kconfig]: This exists on esp32 processors and should not be removed.

Comment (embeddedt) [reply Kconfig]: That's correct; it [instructs the ESP32 build system](https://github.com/lvgl/lvgl/blob/7d3a6c8de4303f69582a3939d36405a06b376ed0/CMakeLists.txt#L31-L33) to set `LV_ATTRIBUTE_FAST_MEM`, which [does exist](https://github.com/lvgl/lvgl/blob/7d3a6c8de4303f69582a3939d36405a06b376ed0/lv_conf_template.h#L281).

Comment (xiaoxiang781216) [reply Kconfig]: Ok, I will remove this patch from PR.

**Comment by kisvegabor** [comment]
Thank you @xiaoxiang781216 

@liebman @C47D do you agree to merge this?

File: Kconfig (hunk 1 above)
Comment (C47D) [reply Kconfig]: Maybe we can add a comment about that on the Kconfig file (the option being used for ESP32 processors)

File: Kconfig [hunk 2]
```diff
@@ -893,16 +897,16 @@ menu "LVGL configuration"
         config LV_TXT_LINE_BREAK_LONG_PRE_MIN_LEN
             int "Min num chars before break"
             default 3
+            depends on LV_TXT_LINE_BREAK_LONG_LEN > 0
```
Comment (C47D) [inline Kconfig:902]: I didn't knew you can use the > operator in here, it's nice to know.

File: Kconfig [hunk 3]
```diff
@@ -129,7 +129,7 @@ menu "LVGL configuration"
         default "free"
         depends on LV_MEM_CUSTOM
 
-    config LV_MEM_SIZE_BYTES
+    config LV_MEM_SIZE_KILOBYTES
```
Comment (C47D) [inline Kconfig:132]: Thanks for updating the name, it's much better now.

**Comment by C47D** [comment]
Hi @xiaoxiang781216 , thanks for the improvements, did you used any script to format the Kconfig file (remove trailing white spaces and replace tabs with spaces)?

**Comment by liebman** [comment]
This looks good to me.

**Comment by xiaoxiang781216** [comment]
> Hi @xiaoxiang781216 , thanks for the improvements, did you used any script to format the Kconfig file (remove trailing white spaces and replace tabs with spaces)?

vscode has commands to remove the space and convert the tab.

**Comment by C47D** [comment]
Thanks for the info, all looks good to me.

**Comment by kisvegabor** [comment]
I merge it then. Thank you!

**Pull request merged**